src/PureCloud.Client/Apis/LanguageUnderstandingApi.cs

```diff
@@ -0,0 +1,532 @@
+        
+        response.EnsureSuccessStatusCode();
+
+        return await response.Content.ReadFromJsonAsync<NluDomain>(_options.JsonSerializerOptions, cancellationToken);
+    }
+
+    /// <inheritdoc />
+    public async Task<NluFeedbackListing> GetLanguageUnderstandingDomainFeedbackAsync(string domainId, string intentName = null, string assessment = null, string dateStart = null, string dateEnd = null, bool? includeDeleted = null, string language = null, int? pageNumber = null, int? pageSize = null, bool? enableCursorPagination = null, bool? includeTrainingUtterances = null, string after = null, IEnumerable<string> fields = null, CancellationToken cancellationToken = default)
+    {
+        ArgumentException.ThrowIfNullOrEmpty(domainId);
+
+        var parameters = new NameValueCollection();
+        if (!string.IsNullOrEmpty(intentName))
+            parameters.Add("intentName", intentName);
+        if (!string.IsNullOrEmpty(assessment))
+            parameters.Add("assessment", assessment);
+        if (!string.IsNullOrEmpty(dateStart))
+            parameters.Add("dateStart", dateStart);
+        if (!string.IsNullOrEmpty(dateEnd))
+            parameters.Add("dateEnd", dateEnd);
+        if (includeDeleted.HasValue)
+            parameters.Add("includeDeleted", includeDeleted.Value.ToString());
+        if (!string.IsNullOrEmpty(language))
+            parameters.Add("language", language);
+        if (pageNumber.HasValue)
+            parameters.Add("pageNumber", pageNumber.Value.ToString());
+        if (pageSize.HasValue)
+            parameters.Add("pageSize", pageSize.Value.ToString());
+        if (enableCursorPagination.HasValue)
+            parameters.Add("enableCursorPagination", enableCursorPagination.Value.ToString());
+        if (includeTrainingUtterances.HasValue)
+            parameters.Add("includeTrainingUtterances", includeTrainingUtterances.Value.ToString());
+        if (!string.IsNullOrEmpty(after))
+            parameters.Add("after", after);
+        if (fields != null)
+            parameters.Add("fields", string.Join(",", fields));
+
+        var uri = UriHelper.GetUri($"/api/v2/languageunderstanding/domains/{domainId}/feedback", parameters);
+        var response = await _httpClient.GetAsync(uri, cancellationToken);
+        
+        response.EnsureSuccessStatusCode();
+
+        return await response.Content.ReadFromJsonAsync<NluFeedbackListing>(_options.JsonSerializerOptions, cancellationToken);
+    }
+
+    /// <inheritdoc />
+    public async Task<NluFeedbackResponse> GetLanguageUnderstandingDomainFeedbackAsync(string domainId, string feedbackId, IEnumerable<string> fields = null, CancellationToken cancellationToken = default)
+    {
+        ArgumentException.ThrowIfNullOrEmpty(domainId);
+        ArgumentException.ThrowIfNullOrEmpty(feedbackId);
+
+        var parameters = new NameValueCollection();
+        if (fields != null)
+            parameters.Add("fields", string.Join(",", fields));
```

Comment:
Add brackets and don't glue the values like this. Instead add them to the NameValueCollection